osu.Game/Users/Country.cs

```diff
@@ -45,6 +46,9 @@ public string FlagName
         [BackgroundDependencyLoader]
         private void load(TextureStore ts)
         {
+            if (ts == null)

```

Comment:
i guess it's fine to have these, but they aren't really necessary as `BackgroundDependencyLoader` will always return non-null when it hasn't been added with the `allowNulls:true` parameter.